spec/controllers/users_controller_spec.rb

```diff
@@ -28,4 +28,18 @@
       expect(user.reload.name).to eq(user.name)
     end
   end
+
+  describe 'DELETE users#destroy' do
+    it 'deletes the user' do
+      user = create(:user)
+      delete :destroy, params: { id: user.id }
+      expect(response).to have_http_status(:ok)
+      expect { user.reload }.to raise_error(ActiveRecord::RecordNotFound)
```

Comment:
This works but isn't quite the approach we want to take. Take a look at some of the other tests to see how we do it

Reply:
Agreed - I'm sorry, I should have checked before.